bee-message/src/error/validation.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    address::Address,
+    input::UtxoInput,
+};
+
+use crypto::Error as CryptoError;
+
+use alloc::string::String;
+use core::fmt;
+
+#[derive(Debug)]
+pub enum ValidationError {
```

Comment:
I'd name every field of every variant with more than one field to be sure we know which one is the expected/found value. But that's a personal preference.

Reply:
Yeah, I think I'm inclined to agree (so far this is a hold-over from the error enum of the old crate).

Reply:
I agree with Christian.

Reply:
Done :+1: